src/Resources/app/administration/src/vite.config.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+    resolve: {
+        alias: {
+            SwagExtensionStore: __dirname
```

Comment:
Is this pointing to the src/ dir globally (relative to the vite.config), or is this rather an alias for "./" ?

Reply:
`__dirname` is a commonjs alias for the current directory this file lives in -> `src/Resources/app/administration/src`